physx-sys/src/physx_api.cpp

```diff
@@ -118,6 +118,32 @@ class RaycastFilterCallback : public PxQueryFilterCallback
     }
 };
 
+typedef int (*RaycastHitCallback)(const PxRigidActor *actor, PxFilterData const *filterData, const void *userData);
```

Comment:
I think it might be a good idea to have all the parameters that `PxQueryFilterCallback::preFilter` receives, so that if we find out in a while that we need to use the `PxShape` or `PxHitFlags`, we don't end up needing to create another callback type.

Reply:
Yeah, fair enough.